### ISSUE TITLE
I need only bbox of my own datasets by using model. Simple way please~

### ISSUE THREAD

**Issue by danke1** [facebookresearch/maskrcnn-benchmark#1320]
## ❓ Questions and Help
Only what I need is to get a bbox of my own datasets by using your Mask r-cnn model pre-trained on COCO datasets so as to get a human centered cropped image that will feed into another model as inputs for next step.

There are so much imformation to me..

Can you guide me in a simple & direct way?

How can I get a pretrained model on COCO datasets and how can i change the code for using my own datasets..! Reference is also okay plz help me !

I'm hoping not to download COCO datasets because it will be a lot of memory..
